string_lab.js

```diff
@@ -0,0 +1,155 @@
+// 1. `console.log` the first character of a string.
```

Comment:
Overall good demonstration of string properties and methods. I like how you included loops to practice with them. Keep up the good work and keep doing bonus questions!